src/users/users.service.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { UsersService } from './users.service';
+import { User } from './user.model';
+import { getModelToken, SequelizeModule } from '@nestjs/sequelize';
+import * as consts from '../../src/test.constants';
+import { Repository } from 'sequelize-typescript';
+
+describe("UsersService Unit Tests", () => {
```

Comment:
Needs negative testing for find methods similar to the create/remove/update methods.